### ISSUE TITLE
Safari: Missing Visual Focus on Update Password Form

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4314]
![Screen Shot 2019-08-29 at 2 09 07 PM](https://user-images.githubusercontent.com/15113219/63965287-cc070d80-ca66-11e9-8bfb-ec94cf888bb7.png)

missing focus on input fields and buttons

**Comment by Zeyber** [comment]
Focus is visible on forms. Hard to see on buttons on chrome.

**Comment by KateChuen** [comment]
Closing as duplicate of #4334
